packages/cryptoplease/lib/features/outgoing_direct_payments/src/bl/odp_request.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:solana/solana.dart';
+
+import '../../../../core/amount.dart';
+
+part 'odp_request.freezed.dart';
+
+@freezed
+class ODPRequest with _$ODPRequest {
+  const factory ODPRequest({
```

Comment:
What is the difference between the existing `PaymentRequest` and this one?

Reply:
`PaymentRequest` is the entity created when you create a payment request on the main wallet screen, generating a Solana Pay Link to be shared. The `ODPRequest` is created when you open a Solana Pay Link, allowing the app to track the payment status when you send money.

Reply:
But in that case, it's just an `OutgoingDirectPayment`.

Reply:
Removed in the last update, as discussed.